assets/styles/global.scss

```diff
@@ -90,6 +90,13 @@ button {
             padding: 0 $whitespace-s-sides + px;
         }
     }
+    .about-results {
```

Comment:
should we call this search-results so it is more re-usable naming wise?

Reply:
i named it about-results because this section isn't the results themselves, but rather a description of the results